attestation-service/src/rvps/mod.rs

```diff
@@ -31,7 +32,14 @@ pub mod grpc;
 pub mod builtin;
 
 fn default_store_type() -> String {
-    DEFAULT_STORAGE_TYPE.into()
+    #[cfg(feature = "rvps-builtin")]
+    {
+        DEFAULT_STORAGE_TYPE.into()
+    }
+    #[cfg(not(feature = "rvps-builtin"))]
+    {
+        return "LocalFs".into()
+    }
```

Comment:
unclear what the correct thing to do is here

Reply:
hmm I think the best solution here would be to get rid of these default functions entirely and change the config struct to have Optional fields. Then we can use `unwrap_or` to swap in the default values as needed (i.e. inside of the RVPS module). That's slightly more work but I think it will untangle the dependencies.

Reply:
The configuration part about RVPS seem not good here. If `remote_addr` is set, the `store_type` is actually not useful. It might be better to not have useless fields that can be set. Let me make a fix to cover this. I made a PR https://github.com/confidential-containers/trustee/pull/553

Reply:
I would have settled for a simpler fix but #553 seems good at first glance.